assets/sass/v2/_beacon.scss

```diff
@@ -1,7 +1,7 @@
 .siw-main-header {
   position: relative;
   .beacon-container {
-    top: -160px;
+    top: -183px;
```

Comment:
how we decide this number? will it be trouble on different screen sizes ?

Reply:
givent that it was already hardcoded, no?

Reply:
would be better if have percentage here..

Reply:
The container this beacon sits in is not responsive. The beacon is absolutely positioned inside a relative container so it's not going to change with different screen resolutions.
As for different logo heights in the header, if the beacon was percentage positioned in a fixed height container that would place the beacon at a different vertical position with every unique logo height. This css means it wouldn't guarantee the beacon was always vertically centered to the bottom edge of the header.